internal/clients/team4/utilities.go

```diff
@@ -19,6 +19,13 @@ func (c *client) getSeason() uint {
 	return 0
 }
 
+func (c *client) getTurnLength(role shared.Role) uint {
+	if c.ServerReadHandle != nil {
+		return c.ServerReadHandle.GetGameConfig().IIGOClientConfig.IIGOTermLengths[role]
+	}
+	return 1
```

Comment:
Maybe return 0, so we know it was not called for whatever reason?

Reply:
ok